src/js/providers/html5.ts

```diff
@@ -444,6 +451,14 @@ function VideoProvider(this: HTML5Provider, _playerId: string, _playerConfig: Ge
         });
     };
 
+    function _canPlayHandler(): void {
+        _canSeek = true;
```

Comment:
I think the only change significant to the issue we're trying to workaround is deffereing `_canSeek = true` a little bit. `canplay` may be too early, but using "canplaythrough" is not wise.  `_canSeek` is also automatically flipped to true if we have a seekable end, so maybe we only need to make a new event handler to set it later (not change anything else about canplay) and maybe we can use another event (like play or another loaded event).